Sources/OrderedCollections/OrderedDictionary/OrderedDictionary+Sendable.swift

```diff
@@ -0,0 +1,15 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Collections open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
```

Comment:
2022?

Reply:
Oh how time flies when we're having fun!